.travis.yml

```diff
@@ -10,7 +10,6 @@ branches:
 language: php
 
 php:
-  - '7.1'
```

Comment:
You need to update the dev-kit too, otherwise this will be reverted with the next cron run

Reply:
The 3.x branch is not on dev-kit anymore: https://github.com/sonata-project/dev-kit/blob/master/config/projects.yml#L105